src/cuda.cpp

```diff
@@ -142,6 +142,10 @@ FastEvent::~FastEvent() {
 }
 
 void FastEvent::record(cudaStream_t stream) {
+/* This block is gated by STREAM_MEM_OP support because hip does not define
```

Comment:
Is it possible that this pushes one call down the events path and a different `FastEvent` method down the stream memory operation path?

Reply:
Fixed.